client/blocks/reader-post-options-menu/index.jsx

```diff
@@ -232,19 +232,19 @@ class ReaderPostOptionsMenu extends React.Component {
 
 					{ isBlockPossible && (
 						<PopoverMenuItem onClick={ this.blockSite }>
-							{ translate( 'Block Site' ) }
+							{ translate( 'Block site' ) }
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.